src/Social/Contract/Interfaces/UserInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Packages\Social\Contract\Interfaces;
```

Comment:
@arfenis we don't have to say contracts Interface o Interfaces since we have that on their name, it's better to use the Folder of the name of whatever the interface belongs to